data/management/commands/fetch_external_data.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import os.path
+
+from django.core.management.base import BaseCommand
+import requests
+
+from community.git import get_org_name
+
+
+class Command(BaseCommand):
+    help = 'Fetch external data'
+
+    def add_arguments(self, parser):
+        parser.add_argument('output_dir', nargs='?', type=str)
+        parser.add_argument('repo_name', nargs='?', type=str)
+        parser.add_argument('filenames', nargs='+', type=str)
+        parser.add_argument('--allow_failure', action='store_true',
+                            default=False, help='Don\'t raise exceptions.')
+
+    def handle(self, *args, **options):
+        logger = logging.getLogger(__name__)
+        output_dir = options.get('output_dir')
+        repo_name = options.get('repo_name')
+        filenames = options.get('filenames')
+        allow_failure = options.get('allow_failure')
+        org_name = get_org_name()
+
+        for filename in filenames:
+            url = ('http://' + org_name + '.github.io/'
```

Comment:
please enhance `community.git` , so that you are re-using logic of `get_deploy_url`/`get_upstream_deploy_url`.  Probably this can be done by adding args to `get_upstream_deploy_url`, or maybe a new function is needed.